frontend/javascripts/oxalis/view/left-border-tabs/mapping_settings_view.tsx

```diff
@@ -170,17 +185,18 @@ class MappingSettingsView extends React.Component<Props, State> {
 
     // The mapping toggle should be active if either the user clicked on it (this.state.shouldMappingBeEnabled)
     // or a mapping was activated, e.g. from the API or by selecting one from the dropdown (this.props.isMappingEnabled).
-    const shouldMappingBeEnabled = this.state.shouldMappingBeEnabled || this.props.isMappingEnabled;
+    const shouldMappingBeEnabled = this.state.shouldMappingBeEnabled || isMappingEnabled;
     const renderHideUnmappedSegmentsSwitch =
-      (shouldMappingBeEnabled || this.props.isMergerModeEnabled) &&
-      this.props.mapping &&
-      this.props.hideUnmappedIds != null;
-    const isDisabled = this.props.isEditableMappingActive || this.props.isMappingLocked;
-    const disabledMessage = this.props.isEditableMappingActive
-      ? "The mapping has been edited through proofreading actions and can no longer be disabled or changed."
-      : this.props.mapping
-        ? "This mapping has been locked to this annotation, because the segmentation was modified while it was active. It can no longer be disabled or changed."
-        : "The segmentation was modified while no mapping was active. To ensure a consistent state, mappings can no longer be enabled.";
+      (shouldMappingBeEnabled || isMergerModeEnabled) && mapping && hideUnmappedIds != null;
+    // TODO: Ask whether changing the mapping should be disabled when the annotation is locked.
```

Comment:
i would say it should not be allowed, because the active mapping is stored in the annotation. changing it would require storing that update (and that is not allowed).